docs/UserGuide.md

```diff
@@ -393,25 +407,25 @@ Format: `sell upc/[UPC] qty/[Quantity]`
 
 **REQUIRED** Parameters:
 
-* The `upc/` parameter whereby `[UPC]` refers to the identification number assigned to the item at the point 
+* The `[UPC]` field refers to the identification number assigned to the item at the point 
 of **initial addition** of the item. `[UPC]` has to be valid, that is, it **EXISTS** in the database, and has to be 
 a **POSITIVE NUMBER** and **NOT EMPTY**. Do **NOT** include `[` and `]` in the input.
-* The `qty/` parameter whereby `[Quantity]` refers to the amount of stock to be **DEDUCTED** from the current 
+* The `[Quantity]` field refers to the amount of stock to be **DEDUCTED** from the current 
 stock levels recorded. `[Quantity]` input **SHOULD NOT** be **EMPTY**,a **NEGATIVE INTEGER**, **ZERO** or a **STRING**. 
 Do **NOT** include `[` and `]` in the input.
```

Comment:
For both the sell and restock ug there might be merge conflicts, I have deleted the wordy parts of the upc and quantity and shifted it below into the form of a warning(!>)

Reply:
can can, ill resolve it accordingly after you have merged